reorder_python_imports.py

```diff
@@ -551,13 +556,16 @@ def _add_future_options(parser):
 def _future_removals(args):
```

Comment:
probably should rename this function now -- also we probably want to rename the help texts for the `--pyXX-plus` options since they say "future imports" in them, but that hasn't been true since the six removals were added

Reply:
I wonder if we should merge `io.open` with the `six` removals instead of with the `__future__` removals -- would probably make the code change smaller

or maybe we should combine them all?

Reply:
Good point. I didn't consider six removals when I wrote this. Can try to look at combining them in the next few days